npm/pkg/dataplane/ipsets/ipsetmanager_linux.go

```diff
@@ -118,10 +118,9 @@ func (iMgr *IPSetManager) resetIPSets() error {
 }
 
 // this needs to be a separate function because we need to check creator contents in UTs
-func (iMgr *IPSetManager) fileCreatorForReset(ipsetListOutput []byte) (creator *ioutil.FileCreator, originalNumAzureSets int, destroyFailureCount *int) {
-	zero := 0
-	destroyFailureCount = &zero
-	creator = ioutil.NewFileCreator(iMgr.ioShim, maxTryCount, ipsetRestoreLineFailurePattern)
+func (iMgr *IPSetManager) fileCreatorForReset(ipsetListOutput []byte) (*ioutil.FileCreator, int, *int) {
```

Comment:
It seems we do not return pointer.
```go
(*ioutil.FileCreator, int, *int) -> (*ioutil.FileCreator, int, int)
```
and can use non-pointer operation in caller as well.